pkg/apis/core/v1alpha1/types_cloudprofile.go

```diff
@@ -83,6 +83,19 @@ type CloudProfileSpec struct {
 	VolumeTypes []VolumeType `json:"volumeTypes,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 }
 
+// SeedSelector describes constraints for selecting seeds by the scheduler for shoots based on the used profile
+type SeedSelector struct {
+	// LabelSelector is optional and can be used to select seeds by their label settings
+	// +optional
+	*metav1.LabelSelector `json:",inline,omitempty"`
+	// Providers is optional and can be used by restricting seeds by their provider type. '*' can be used to enable seeds regardless of their provider type.
+	// +optional
+	Providers []string `json:"providers,omitempty"`
+	// Seeds is optional and can be used by restricting seeds by their name. '*' can be used to enable seeds regardless of their name.
+	// +optional
+	Seeds []string `json:"seeds,omitempty"`
```

Comment:
What would be the usecase of selecting Seeds by their name?
This seems cumbersome, I cannot think about a place in kubernetes where you select something by its name.